backend/configs/usr/lib/cgi-bin/factory_reset.py

```
@@ -0,0 +1,51 @@
+
+    # open/close it for writing to trigger wb-watch-update's special mode
+    USE_FACTORY_FIT_FLAG = os.path.join(RW_DIR, "wb_use_factory_fit.flag")
+    with open(USE_FACTORY_FIT_FLAG, "ab") as fp:
+        pass
+
+    sys.stdout.write("Status: 200\r\n\r\n")
+
+
+try:
+    main()
+except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)